### PR TITLE
Set the height and width on the SVG root

### PR DESCRIPTION
**Pull request by josephspens**
The `svg-pan-zoom` tool was apparently [stripping dimensions from the SVG root](https://www.npmjs.com/package/svg-pan-zoom#svg-height-is-broken), resulting in the behavior described in [SHEL-175](https://ticketevolution.atlassian.net/browse/SHEL-175).

The solution is to copy the viewbox values onto the width/height properties when we set the map up.

### CODE REVIEW

File: src/ticketmap/types.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export interface SectionZoneMapping {
 export interface DefaultProps {
     isZoneDefault: boolean
     showLegend: boolean
-    mapFontFamily?: string
+    mapFontFamily: string
```
Comment (mayavera) [inline src/ticketmap/types.ts]: This seems out of scope (along with the other change to font logic)

File: src/ticketmap/index.tsx [hunk 2]
```diff
@@ -151,8 +152,9 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
 
   setupMap() {
     const mapSvg = this.mapRootRef.querySelector('svg') as SVGSVGElement
-    mapSvg.style.width = '100%'
-    mapSvg.style.height = '100%'
+    const [x, y, width, height] = mapSvg.getAttribute('viewBox').split(' ')
```
Comment (mayavera) [inline src/ticketmap/index.tsx]: Do all of our maps have `viewBox` attributes?

Comment (josephspens) [reply src/ticketmap/index.tsx]: Yes they do
